graders/python/python_autograder/pl_unit_test.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+from types import FunctionType
+from collections import namedtuple
+from pl_helpers import (points, name, save_plot, print_student_code,
+                        not_repeated)
+from pl_execute import execute_code
+from code_feedback import Feedback
+
+# Needed to ensure matplotlib runs on Docker
+import matplotlib
+matplotlib.use('Agg')
+
+
+class PrairieLearnTestCase(unittest.TestCase):
+
+    include_plt = False
+    student_code_string = 'test_print_student_code'
+    student_code_file = 'user_code.py'
+    iter_num = 0
+    total_iters = 1
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Thoughts on doc strings for functions here?

Reply:
Yes, can add those.